src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public static final int MAX_NUMBER_BALL = 100;
+    private static final Random random = new Random();
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public static Ball getRandomBall() {
```

Comment:
don't use static methods without real need to call them without class instance

![](https://user-images.githubusercontent.com/20326858/224994214-ad74056c-798d-4c68-865e-f3e0eaa84b07.png)